examples/ZMQPP/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+cmake_minimum_required(VERSION 3.2)
+
+# Emulate HunterGate:
+# * https://github.com/hunter-packages/gate
+include("../common.cmake")
+
+project(download-ZMQPP)
+
+hunter_add_package(ZMQPP)
+
+find_package(ZMQPP CONFIG REQUIRED)
+
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
```

Comment:
> I added the following lines to the example CMakelists.txt, before hunter_add_package

> I got the c++11 missing error message from the ZMQPP's CMakelists.txt

Setting C++ flags locally in project doesn't affect C++ flags for 3rd party packages from Hunter. Toolchain should be used. As a workaround you can set `c++11` manually but protect it with `if(no_toolchain)` condition.